index.html

```diff
@@ -1705,6 +1706,70 @@ <h3>Validity Period</h3>
 valid from the time the <code>verifiable credential</code> was created.
         </p>
 
+        <h4>Representing Time</h4>
+
+        <p>
+When representing time values, implementers are urged to understand that times
+are not implemented in a uniform manner throughout the world. For example:
+        </p>
+
+        <ul>
+          <li>
+The observation of daylight savings time does not start or end on the same day
```

Comment:
```suggestion
The observation of daylight saving time does not start or end on the same day
```

Reply:
I'll also add: the "tripping hazard" related to time zones and DST/ST for your specification's users is that governments change time zone rules. These changes can result in a given wall time being expressed differently. For example, consider if the zone `America/Los_Angeles`, which has a raw offset of UTC-8 and observes DST decided to stop observing DST in 2024.

A given certificate might have been generated to be:

```json
"validUntil": "2024-07-12T12:00:00-07:00"  // noon on July 12
```

The replacement certificate, created after the change to the time zone might have:

```json
"validFrom": "2024-07-12T12:00:00-08:00" // noon on July 12
```

This leaves a gap of an hour...

Consider adding a note or comment such as:

```suggestion
<p class="note">Time zone definitions are occasionally changed by their governing body. When replacing or issuing new credentials, care should be exercised to ensure that changes to local time zone rules do not result in unexpected gaps in validity.</p>
```

(You could then adapt the example above. I intend to add this example to the "past and future" use case in the timezone note.)